packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.service.js

```diff
@@ -0,0 +1,10 @@
+import 'moment';
+
+export default class PciStoragesColdArchiveService {
+  /* @ngInject */
+  constructor($http, $q, OvhApiCloudProjectUser) {
+    this.$http = $http;
+    this.$q = $q;
+    this.OvhApiCloudProjectUser = OvhApiCloudProjectUser;
+  }
+}
```

Comment:
This service is required to develop Tape project.
It will be used for all jira we have to develop with Rafik.

It's a base service that we have to complete regarding our respective tasks.